prover/prover_cli/src/commands/get_proof_progress.rs

```diff
@@ -0,0 +1,56 @@
+use anyhow::Context as _;
+use clap::Args as ClapArgs;
+use sqlx::{postgres::PgPoolOptions, Row};
+use zksync_config::PostgresConfig;
+use zksync_env_config::FromEnv;
+
+#[derive(ClapArgs)]
+pub(crate) struct Args {
+    #[clap(short, long)]
+    l1_batch_number: i32,
+}
+
+pub(crate) async fn run(args: Args) -> anyhow::Result<()> {
+    let pool = PgPoolOptions::new()
```

Comment:
Also, if we use the prover_dal methods, we probably should use ConnectionPool from db_connection crate